daprdocs/content/en/reference/components-reference/supported-bindings/_index.md

```diff
@@ -76,6 +76,7 @@ Table captions:
 |------|:----------------:|:-----------------:|--------| --------- | ---------- |
 | [Azure Blob Storage]({{< ref blobstorage.md >}})            |    | ✅ | Alpha | v1 | 1.0 |
 | [Azure CosmosDB]({{< ref cosmosdb.md >}})                   |    | ✅ | Alpha | v1 | 1.0 |
+| [Azure CosmosGraphDB]({{< ref cosmosgrapdb.md >}})              |    | ✅ | Alpha | v1 | 1.5 |
```

Comment:
@hokandil, it would be appreciated if you could submit a PR to rename the component to "CosmosDBGremlinAPI" and also update the doc references to reflect that change.

Reply:
If this change is something you can address relatively soon, we can still aim to get this into the v1.5 release.

Reply:
@hokandil - We will need you to change the name of the component in the next day if this is going into the v1.5 release for next week. Otherwise this will be moved the the v1.6 release in Jan. Thanks